config/consensus.go

```diff
@@ -1146,11 +1155,15 @@ func initConsensusProtocols() {
 
 	vFuture.LogicSigVersion = 7
 	vFuture.MinInnerApplVersion = 4
-
 	vFuture.UnifyInnerTxIDs = true
 
 	vFuture.EnableSHA256TxnCommitmentHeader = true
 
+	// Boxes (unlimited global storage)
```

Comment:
@jannotti Optional - When considering the initial Box consensus configuration values, I thought future maintainers may ask:  What guiding rationale led to defining these values?

If you feel there's a few sentences of context that enriches maintainer comprehension, then I think an inline comment is beneficial.   Otherwise, the thread can be resolved.

Reply:
Resolving - We agreed to review in subsequent PRs prior to merging to `master.`